static/js/place/overview.tsx

```diff
@@ -32,18 +32,26 @@ interface OverviewPropType {
 
 class Overview extends React.Component<OverviewPropType> {
   render(): JSX.Element {
+    // only show ranking for places in the US, but not the US itself.
+    const shouldShowRanking = this.props.dcid.startsWith("geoId/");
     return (
       <section className="factoid col-12">
         <div className="row">
-          <div className="col-12 col-md-4">
+          <div
+            className={`col-12 ${
+              shouldShowRanking && "col-md-4 map-with-padding"
```

Comment:
nit: I'm not sure "map-with-padding" is the best class to add here because this class will be added for maps on place overview page and wherever else Overview is rendered. However, only the nl interface will actually have padding for maps with this class. 

Maybe instead, we can add a "overview-with-ranking" class to line 38 & in the nl_interface.scss, we can do `.overview-with-ranking .map-container { margin-right: 15px }`

Reply:
Switched to using overview-with-ranking as you suggested! Didn't see any changes to the screenshots in my testing.